spec/services/activitypub/process_account_service_spec.rb

```diff
@@ -1,5 +1,26 @@
 require 'rails_helper'
 
 RSpec.describe ActivityPub::ProcessAccountService do
-  pending
+  subject { described_class.new }
+
+  context 'property values' do
+    let(:payload) do
+      {
+        id: 'https://foo',
+        type: 'Actor',
+        inbox: 'https://foo/inbox',
+        attachment: [
+          { type: 'PropertyValue', name: 'Pronouns', value: 'They/them' },
+          { type: 'PropertyValue', name: 'Occupation', value: 'Unit test' },
+        ],
+      }.with_indifferent_access
+    end
+
+    it 'parses out of attachment' do
+      account = subject.call('alice', 'example.com', payload)
+      expect(account.fields).to be_a Hash
+      expect(account.fields).to include('name' => 'Pronouns', 'value' => 'They/them')
+      expect(account.fields).to include('name' => 'Occupation', 'value' => 'Unit test')
```

Comment:
is it worth checking that `account.fields` doesn't have more than the payload declared?